libs/barista-components/stacked-series-chart/src/stacked-series-chart.spec.ts

```diff
@@ -598,6 +606,16 @@ describe('DtStackedSeriesChart', () => {
         const axis = getSeriesAxis();
         expect(axis).toBeTruthy();
       });
+
+      it('should compact the labels', () => {
+        rootComponent.labelAxisMode = 'compact';
+        fixture.detectChanges();
+
+        const compactedLabels = getAllTrackLabelsCompacted();
```

Comment:
If changing the compacted class to the root, you should change this test and simply check it in one place